examples/LaTeX Circuits.ipynb

```diff
@@ -0,0 +1,204 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyquil.gates import *\n",
+    "from pyquil.quil import Program\n",
+    "from pyquil.latex import to_latex, display"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Basic Functionality: LaTeX generation and inline IPython display\n",
+    "\n",
+    "There are two main functions exposed: \n",
+    "- `to_latex` produces LaTeX source code for producing figures.\n",
+    "- `display` renders to an `IPython.display.Image` object. This requires that `pdflatex` and Imagemagick's `convert` are installed and available in the user's path."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\\documentclass[convert={density=300,outext=.png}]{standalone}\n",
+      "\\usepackage[margin=1in]{geometry}\n",
+      "\\usepackage{tikz}\n",
+      "\\usetikzlibrary{quantikz}\n",
+      "\\begin{document}\n",
+      "\\begin{tikzcd}\n",
+      "\\lstick{\\ket{q_{0}}} & \\gate{X} & \\ctrl{1} & \\qw & \\qw \\\\\n",
+      "\\lstick{\\ket{q_{1}}} & \\qw & \\targ{} & \\gate{H} & \\qw\n",
+      "\\end{tikzcd}\n",
+      "\\end{document}\n"
+     ]
+    }
+   ],
+   "source": [
+    "prog = Program(\"X 0\\n CNOT 0 1\\n H 1\")\n",
```

Comment:
use `+=` to build up programs -- if its annoying to use `+=` then we should open an issue to fix whatever your pain points are (maybe the pragma and dagger?)

Reply:
I've changed the first two examples to avoid using escape characters etc. Those are built using the gate objects, e.g. `Program(X(0), CNOT(0,1), H(1))`. For the last example I'm going to keep it as a big multiline string -- this is how it came to me (I generated it from the compiler), and I don't think anything would be improved by changing this.